### ISSUE TITLE
spack installation can not compile plugin

### ISSUE THREAD

**Issue by tjhei** [geodynamics/aspect#4501]
Verify spack installation and how to compile plugins. User report 
```
$HOME/build/aspect/benchmarks/blankenbach/plugin/code.cc:20:10: fatal error: aspect/global.h: No such file or directory
20 | #include <aspect/global.h>
```
on the forum at https://community.geodynamics.org/t/beginner-question-error-running-blankenbach-benchmark-cant-find-global-h/2269

related to #4498 and #4500

**Comment by tjhei** [comment]
I can confirm that ``spack install aspect@develop`` works, while ``spack install aspect`` (v 2.3.0) generates the error in #4498. I am not sure why the reported error is for global.h, though. Regardless, we need to switch to the development version because of #4498.

I added a note to #4380

**Comment by bangerth** [comment]
What confuses me is how cmake reports that ASPECT was found, but then the header file can't be found. Does spack install the ASPECT.cmake file in a different location relative to global.h than happens otherwise? Separately, I don't think I've every "installed" aspect. I always just build it and let my `ASPECT_DIR` point to the build directory. Might that be the issue?

**Comment by tjhei** [comment]
> Does spack install the ASPECT.cmake file in a different location relative to global.h than happens otherwise?

It is also not clear to me why global.h is missing for them. No, I tried it out.

> Separately, I don't think I've every "installed" aspect. I always just build it and let my `ASPECT_DIR` point to the build directory. Might that be the issue?

Our latest release (version 2.3) did not work when installed, we fixed that in #4195. So, yes, this is also definitely an issue.

**Comment by tjhei** [comment]
I will close this issue as I did not hear about the problem again. If anyone can reproduce this issue, feel free to comment here...
